src/providers/github/Github.ts

```diff
@@ -91,11 +113,14 @@ export class Github {
    */
   async createTag(tag: string, sha: string): Promise<void> {
     try {
-      await this.octokit.git.createRef({
+      await this.octokit.request("POST /repos/{owner}/{repo}/git/refs", {
         owner: this.owner,
-        repo: this.repo,
+        repo: "DAppNodePackage-SSV-holesky",
```

Comment:
Is this a typo? shouldnt be `this.repo`?

Reply:
Yes, sorry. My mistake after testing